src/hooks/useCourtStats.js

```diff
@@ -0,0 +1,126 @@
+import { useMemo } from 'react'
+import { useTotalANTStakedPolling } from './useCourtContracts'
+import { useActiveJurorsNumber } from '../hooks/query-hooks'
+import {
+  useJurorRegistrySubscription,
+  useTotalRewardsSubscription,
+} from '../hooks/subscription-hooks'
+import { getKnownToken } from '../utils/known-tokens'
+import { bigNum } from '../lib/math-utils'
+import IconANJ from '../assets/IconANJ.svg'
+import IconANT from '../assets/IconANT.svg'
+import IconDAI from '../assets/IconDAI.svg'
+
+const COURT_STATS = [
+  {
+    label: 'Total Active ANJ',
+    token: { ...getKnownToken('ANJ'), icon: IconANJ },
+  },
+  {
+    label: 'Total Staked ANT',
+    token: { ...getKnownToken('ANT'), icon: IconANT },
+  },
+  { label: 'Total Active Jurors' },
+  {
+    label: 'Total Rewards DAI',
+    token: { ...getKnownToken('DAI'), icon: IconDAI },
+  },
+]
+
+export function useTotalActiveBalance() {
+  const { data: jurorRegistryStats, error } = useJurorRegistrySubscription()
+
+  return useMemo(() => {
+    if (!jurorRegistryStats || error) {
+      return [bigNum(-1), error]
+    }
+    return [bigNum(jurorRegistryStats.totalActive), error]
+  }, [error, jurorRegistryStats])
+}
+
+function useTotalRewards() {
+  const { data: rewards, error } = useTotalRewardsSubscription()
+
+  return useMemo(() => {
+    if (!rewards || error) {
+      return [bigNum(-1), error]
+    }
+    return [
+      rewards.reduce(
+        (totalAcumulator, reward) => totalAcumulator.add(reward.amount),
+        bigNum(0)
+      ),
+      error,
+    ]
+  }, [error, rewards])
+}
+/**
+ * Hook to get the dashboard stats ANJ active balance, ANT total stake and the active jurors number
+ * @returns {Array} First item an array with the stats and the second one a loading state
+ */
+function useCourtStats() {
+  const timeout = 5000
```

Comment:
Let's lift this out to be a constant at the module level.

It might also be fine up'ing it to 15s, since block times are generally around that length!

Reply:
sure!